src/overlays/actors/ovl_En_Niw/z_en_niw.h

```diff
@@ -5,11 +5,97 @@
 
 struct EnNiw;
 
+typedef void (*EnNiwActionFunc)(struct EnNiw*, GlobalContext*);
+
+typedef struct {
+    // in OOT this is "type" and there is one instance of an if
+    //  statement specifying `if (true) if (enabled == 1)`
+    //  so in beta there might have been a third value
+    /* 0x0000 */ u8 enabled;
+    /* 0x0004 */ Vec3f pos;
+    /* 0x0010 */ Vec3f vel;
+    /* 0x001C */ Vec3f accel;
+    /* 0x0028 */ s16 life;
+    /* 0x002A */ s16 zRotStart;
+    /* 0x002C */ f32 scale;
+    /* 0x0030 */ f32 zRot;
+    /* 0x0034 */ u8 timer;
+} EnNiwFeather; // size = 0x38
+
+#define ENNIW_LIMBCOUNT 0x10
+
 typedef struct EnNiw {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x678];
+    /* 0x144 */ SkelAnime skelanime;
+    /* 0x188 */ Vec3s limbDrawTbl[ENNIW_LIMBCOUNT];
+    /* 0x1E8 */ Vec3s transitionDrawtable[ENNIW_LIMBCOUNT];
+    /* 0x248 */ EnNiwActionFunc actionFunc;
+    /* 0x24C */ s16 unkTimer24C;
+    /* 0x24E */ s16 unkTimer24E;
+    /* 0x250 */ s16 unkTimer250;
+    // 252 is re-used, after thrown is used as a hop timer
+    /* 0x252 */ s16 unkTimer252;
+    /* 0x254 */ s16 unkTimer254; // might be a running timer
+    /* 0x256 */ s16 sfxTimer1; 
+    /* 0x258 */ s16 flutterSfxTimer;
+    /* 0x25A */ s16 unusedTimer25A;
+    /* 0x25C */ s16 yawTimer; // every 70 frames rechecks yawToPlayer
+    /* 0x25E */ s16 unusedTimer25E;
+    /* 0x260 */ s16 unkTimer260; // iframe hit timer?
+    /* 0x264 */ f32 unk264[10]; 
+    /* 0x28C */ s16 unusedCounter28C; // todo: view in-game and see if it gets read out of actor? 
+    /* 0x28E */ s16 unknownState28E;
+    /* 0x290 */ s16 unk290; // attack niw count? never decrements though?
+    /* 0x292 */ s16 unk292; // toggle timer
+    /* 0x294 */ s16 pad294;
+    /* 0x296 */ s16 unk296; // toggle every other frame timer
+    /* 0x298 */ s16 unk298;
+    /* 0x29C */ s16 unk29A; // toggle timer for func_808917F8
+    /* 0x29C */ u16 unk29C; // changed by EnNiw_Trigger
+    /* 0x29E */ s16 unk29E;
+    /* 0x2A0 */ s16 niwStormActive; // bool
```

Comment:
```suggestion
    /* 0x2A0 */ s16 isStormActive;
```
Same as comment above. `isNiwStormActive` is also fine. Up to you!